app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/MapsActivity.java

```diff
@@ -83,16 +95,55 @@ protected void onCreate(Bundle savedInstanceState) {
         setContentView(R.layout.activity_maps);
         fusedLocationClient = LocationServices.getFusedLocationProviderClient((Activity) this);
         FragmentManager manager = this.getFragmentManager();
-        Repository.getInstance().setContext(this);
-        mController = new MapsPresenter(manager);
+        Repository.ourInstance.setContext(this);
+        mController = new MapsPresenter();
 
         SupportMapFragment mapFragment = (SupportMapFragment) this.getSupportFragmentManager().findFragmentById(R.id.map);
         mapFragment.getMapAsync((OnMapReadyCallback) this);
         RecyclerView mRecView = this.findViewById(R.id.recycler_view_maps);
         mController.mRecView = mRecView;
         mController.setDynamicRecyclerView(this);
 
+        mHomeView = this.findViewById(R.id.home_view);
+        mHomeMenu = this.findViewById(R.id.home_menu);
+
         setUpSearch();
+        setUpMenu();
+    }
+
+    public void onHeaderClick(View view) {
```

Comment:
Why is the code inside commented out?

Reply:
Just because it's placeholder code. It could be anything but I figured I'd leave in a placeholder that doesn't function for reference.

Reply:
Please add a `Note (Preston): ...` explaining that this is just placeholder code and what the code inside is supposed to do